.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.9"
+
+install:
+  - pip install pipenv
+  - pipenv install --dev --deploy
+
+script:
+  - make package_vulnerability
+  - make flake
```

Comment:
Worth running vulture here as well?

Reply:
Perhaps travis isn't the best place since we never look at the build if it passes, but it would be good to make it more visible somewhere, I had completely forgotten we had it installed.

Reply:
I've put it in travis now, if it's an issue we can reconsider.
I don't think we've used vulture in anger before, but decided it's well worth a try at the start of a project